File: Modul 2/Bahasa C (Modul 2)/PRAK204-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
int main(){
    printf("input\n");
    float r,t;
    printf("");
    scanf("%lf" "%lf" , &r, &t);

    float phi = 22/7;
    float bejana = phi*r*r;
    float volume = phi*r*r*t;
    float keliling = 2*phi*r;
    float luas = (2*bejana) + (keliling*t);

    printf("\nOutput\n");
    printf("Volume : \n", volume);
    printf("Luas : \n", luas);
    printf("Keliling : \n", keliling);

    return 0;
}

File: Modul 2/Bahasa Python (Modul 2)/PRAK204-2310817320015-AlfisyahRinaFajriati.py
#case 1
print("Input")
r = float (input(" "))
t = float (input(" "))

phi = 22/7
luas_bejana = phi*r*r
volume = phi*r*r*t
luas = (2*luas_bejana)
keliling = 2*phi*r

print("\nOutput")
print(f"volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"keliling = {keliling:.2f}")

#case2
print("\nInput")
r,t = map(float, input().split())

phi = 22/7
luas_bejana = phi*r*r
volume = phi*r*r*t
luas = (2*luas_bejana)
keliling = 2*phi*r

print("\nOutput")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")
